tests/rptest/services/redpanda.py

```diff
@@ -1366,11 +1368,13 @@ def __init__(self,
                      oauth_client_secret,
                      oauth_audience,
                      api_url,
-                     cluster_id='',
+                     cluster_id=None,
```

Comment:
i think this needs to be `''` because it is set in vtools? not sure but if it does break the cloud checks i'll fix it in a follow-up. i see the tests are passing for docker runs of ducktape so i think this PR is safe to merge.